vllm/attention/backends/abstract.py

```diff
@@ -93,6 +93,14 @@ def copy_blocks(
     ) -> None:
         raise NotImplementedError
 
+    @staticmethod
+    @abstractmethod
+    def use_output() -> bool:
```

Comment:
can you directly make this an attribute? why do we need a function here?

Reply:
Done